introduction.md

```diff
@@ -1,3 +1,6 @@
 Hi, I'm Chan.This is for the code school challenge exercise. 
```

Comment:
Since you're becoming a GitHub master, could you also add "Killing history using git rebase" as one of your deadly skills?